bin/runex

```diff
@@ -41,5 +41,5 @@ fi
 "$ex_home"/bin/build
 
 export CLASSPATH="$EXAMPLES_JAR_PATH:$ex_home/conf:$ACCUMULO_HOME/lib/*:$CLASSPATH"
-java "org.apache.accumulo.examples.$1" ${*:2}
+java ${ACCUMULO_JAVA_OPTS} -Dlog4j.configurationFile="$ex_home"/conf/log4j2.properties "org.apache.accumulo.examples.$1" ${*:2}
```

Comment:
These changes probably shouldn't be here. It regresses the previous changes that ensured the log4j file was loaded from the conf/ directory. I'm not sure why this change is needed.

Reply:
`${ACCUMULO_JAVA_OPTS}` needs to be here to pick up the javaagent. You are suggesting that I remove the log4j system property?

Reply:
Removed only the log4j configuration file from the command in b2e0029 and re-tested